### ISSUE TITLE
Volume sound / music property ?

### ISSUE THREAD

**Issue by Jacques-Olivier-Farcy** [thecodingmachine/workadventure#801]
Hi,

As i am using some ambient sounds in my maps, will it be possible to set the volume of a sound with a property ?

**Comment by pizkaz** [comment]
It is possible now. I believe this issue can be closed.

**Comment by Jacques-Olivier-Farcy** [comment]
Hi @pizkaz ,

Do you mean the volume control UI that is opened when playing a sound in a zone ?

I was more asking to have the possibility in property to play the sound with a specific default volume

**Comment by pizkaz** [comment]
Code: https://github.com/thecodingmachine/workadventure/commit/fdbcd98a9acc7b89e225c8294a881b00b4892d23
Docs: https://workadventu.re/map-building/special-zones (bottom of the page)

You're welcome. :-)

**Comment by aresius** [comment]
Great feature but In the documentation I'm missing the property type and range of `audioVolume` (`float` or `int`, exact meaning of the number specified). Could you please add this information? Thanks.

**Comment by pizkaz** [comment]
Unfortunately I can't because I do not have access to that documentation.

`audioVolume` is a float from 0 to 1.0

**Comment by Moini** [comment]
Does this work for anyone? It doesn't work for me. Volume is set to 0.3 as default value for the slider, but the slider still starts at maximum value.

![2021-09-09_00-47](https://user-images.githubusercontent.com/3240233/132595747-74a89ac7-e52d-4694-bc01-da3e01b47bb8.png)

**Comment by Moini** [comment]
Ah, I see... it works, but the slider isn't adjusted when entering the area. 

Also happens when I have turned down the volume entirely in one area, and then go into another area with sound. The slider is at 100%, even if no sound is played.

**Comment by GRL78** [comment]
The audioVolume property exist for this purpose. Documentation here : https://staging.workadventu.re/map-building/special-zones.md#playing-sounds-or-background-music

**Comment by Moini** [comment]
Yep, the bug is with the slider not updating, not the volume setting per se.
